### ISSUE TITLE
Can't save a file after decryption.

### ISSUE THREAD

**Issue by bigdrew** [open-keychain/open-keychain#2278]
As of version 4.9.1 a file cannot be saved after the decryption process finishes.  The app crashes.  Wiping and reinstalling doesn't fix the problem.  However the issue isn't present after downgrading to version 4.8.1 on F-droid.

**Comment by dschuermann** [comment]
Arg thanks. I was able to reproduce sporadically. We will prepare a fix.

```
03-01 15:53:03.269  15509    15509         AndroidRuntime  D  Shutting down VM
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  FATAL EXCEPTION: main
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  Process: org.sufficientlysecure.keychain, PID: 15509
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  java.lang.AbstractMethodError: abstract method "android.view.MenuItem android.support.v4.internal.view.SupportMenuItem.setAlphabeticShortcut(char, int)"
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.MenuInflater$MenuState.setItem(MenuInflater.java:477)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.MenuInflater$MenuState.addItem(MenuInflater.java:541)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.MenuInflater.parseMenu(MenuInflater.java:193)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.MenuInflater.inflate(MenuInflater.java:114)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at com.cocosw.bottomsheet.BottomSheet$Builder.sheet(BottomSheet.java:531)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.DecryptListFragment.displayBottomSheet(DecryptListFragment.java:497)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.DecryptListFragment$DecryptFilesAdapter$1.onLongClick(DecryptListFragment.java:1009)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.View.performLongClickInternal(View.java:6374)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.View.performLongClick(View.java:6332)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.View.performLongClick(View.java:6350)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.view.View$CheckForLongPress.run(View.java:24735)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.os.Handler.handleCallback(Handler.java:790)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.os.Handler.dispatchMessage(Handler.java:99)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.os.Looper.loop(Looper.java:164)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at android.app.ActivityThread.main(ActivityThread.java:6494)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at java.lang.reflect.Method.invoke(Native Method)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
03-01 15:53:03.271  15509    15509         AndroidRuntime  E  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by dschuermann** [comment]
fixed in https://github.com/open-keychain/open-keychain/pull/2283

**Comment by dschuermann** [comment]
fixed in done in b086cd0 and f0e0ed4
